server/jetstream_cluster.go

```diff
@@ -2272,6 +2281,11 @@ func (js *jetStream) processUpdateStreamAssignment(sa *streamAssignment) {
 	sa.consumers = osa.consumers
 	sa.err = osa.err
 
+	// If we detect we are scaling down to 1, non-clustered, and we had a previous node, clear it here.
+	if sa.Config.Replicas == 1 && sa.Group.node != nil {
+		sa.Group.node = nil
```

Comment:
If this was a R>1 and now R1, did we stop monitoring of that stream? Is that done somewhere else?